### PR TITLE
Fixed clang compile error

### PR DESCRIPTION
**Pull request by trenouf**
Change-Id: I5826f529805b67546b645aa5db09772c92433200

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit 2374cbdc814401947c69d8a4508533489cc944f3
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev e8b82e5f183f4cfcf4f7b893295c49d0fc929fa0</li>
<li>SPVGEN: dev 1560d287f779b342e0019499dda85890cb07affc</li>
<li>XGL: dev b5440b6a385dbcb11739f1bdb89a94bbe5cd499f</li>
<li>LLVM: amd-vulkan-dev 97cc33415120ae3ed472b6dd5cb234b74a80bd80</li>
</ul></details>

<details><summary>Build Failed</summary><ul>

/jenkins/workspace/open-source/llpc-github-pr/driver_build/drivers/xgl/test/shadertest/shaderdb/OpImageSample_TestSeparateSampler_lit.frag:28:15: error: SHADERTEST: expected string not found in input

</ul></details>

File: lower/llpcSpirvLowerImageOp.cpp [hunk 1]
```diff
@@ -836,6 +836,8 @@ llvm::Value* SpirvLowerImageOp::LoadImageDescriptor(
                     isNonUniform);
                 break;
             }
+        default:
+            break;
```
Comment (amdrexu) [inline lower/llpcSpirvLowerImageOp.cpp]: break ->
{
    break;
}

Add {} for each case if any one of them uses {}

**Comment by JacobHeAMD** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 2374cbdc814401947c69d8a4508533489cc944f3
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev e8b82e5f183f4cfcf4f7b893295c49d0fc929fa0</li>
<li>SPVGEN: dev 1560d287f779b342e0019499dda85890cb07affc</li>
<li>XGL: dev b5440b6a385dbcb11739f1bdb89a94bbe5cd499f</li>
<li>LLVM: amd-vulkan-dev 97cc33415120ae3ed472b6dd5cb234b74a80bd80</li>
</ul></details>

<details><summary>Build Failed</summary><ul>

/jenkins/workspace/open-source/llpc-github-pr/driver_build/drivers/xgl/test/shadertest/shaderdb/OpImageSample_TestSeparateSampler_lit.frag:28:15: error: SHADERTEST: expected string not found in input

</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 36f325b2a4093ae75cf9ea301049aaa7051700a8
<details open><summary>Driver commits used in build</summary><ul>

<li>PAL: dev e8b82e5f183f4cfcf4f7b893295c49d0fc929fa0</li>
<li>SPVGEN: dev 1560d287f779b342e0019499dda85890cb07affc</li>
<li>XGL: dev b5440b6a385dbcb11739f1bdb89a94bbe5cd499f</li>
<li>LLVM: amd-vulkan-dev 97cc33415120ae3ed472b6dd5cb234b74a80bd80</li>
</ul></details>

<details><summary>Build Failed</summary><ul>

/jenkins/workspace/open-source/llpc-github-pr/driver_build/drivers/llpc/lower/llpcSpirvLowerImageOp.cpp:677:9: error: case label not within a switch statement

</ul></details>

**Comment by trenouf** [comment]
Abandoning as Jacob is going to land this change as part of another change.

**Pull request closed without merging**